app/src/screens/match/MatchCreateSelectLocationScreen.tsx

```diff
@@ -25,15 +25,15 @@ export const MatchCreateSelectLocationScreen: React.FC<MatchCreateSelectLocation
 }) => {
   const [matchLocation, setMatchLocation] = useState<LatLng | null>(null);
 
-  const [mutate, mutation] = useMatchCreateMutation();
+  const mutation = useMatchCreateMutation();
 
   const onPress = async () => {
     if (matchLocation === null) {
       return;
     }
```

Comment:
Dodajmy jeszcze do przycisku prop `disabled={matchLocation === null}`. A najlepiej wyciągnąc to do zmiennej - `const hasSelectedLocation = matchLocation !== null` i później tą zmienną przypisać do propa

Reply:
w sensie state?